packages/trader/src/Modules/Trading/Containers/__tests__/purchase.spec.tsx

```diff
@@ -11,7 +12,7 @@ const default_mock_store = {
     modules: {
         trade: {
             basis: '',
-            contract_type: 'accumulator',
+            contract_type: TRADE_TYPES.ACCUMULATOR as string,
```

Comment:
@kate-deriv If this requires assertion (as string), it means the type of TRADE_TYPES.ACCUMULATOR is not inferred here. Can adding `as const` or types to the constant TRADE_TYPES help? 🤔 So that we don't have to add this assertion.